front/app/components/FilterSelector/Title.tsx

```diff
@@ -98,8 +98,9 @@ const Title = ({
       textColor={textColor}
       role={role}
       aria-haspopup={ariaHasPopup}
+      aria-labelledby="filterSelectorTitle"
     >
-      <Text className="FilterSelectorTitleText" textColor={textColor}>
+      <Text textColor={textColor} id="filterSelectorTitle">
```

Comment:
An accessibility issue discovered (arguably a bit of a false positive after testing it but still..) The text of the dropdown button was not obviously associated with the button itself.

Reply:
Hmm is there a possiblity that this component will be used more than once on the same page? Because then you will have duplicate IDs, which I assume is also an a11y violation. For this reason I think we should avoid hard-coding IDs in reusable components and instead define them through a prop

Reply:
Good point! I will change this 👍